src/components/Heading/Heading.stories.tsx

```diff
@@ -52,10 +53,47 @@ export const Heading1AsHeading4: StoryObj<Args> = {
   },
 };
 
-export const Heading4ColorNeutral: StoryObj<Args> = {
-  args: {
-    children: "Neutral color Heading 4",
-    color: "neutral",
-    size: "h4",
+/**
+ * 1) Insufficient color contrast is expected since we are displaying all color variants on various color backgrounds.
+ */
+export const Variants: StoryObj<Args> = {
+  ...Heading1,
+  render: (args) => {
+    const headings: React.ReactNode[] = [];
+    VARIANTS.forEach((variant) => {
+      if (variant !== "info")
+        headings.push(
+          <Heading {...args} key={`${variant}`} variant={variant}>
+            {variant}
+          </Heading>,
+        );
+    });
+    return (
+      <div className={styles["variant__table"]}>
+        <div className={styles["variant--white"]}>{headings}</div>
+        <div className={styles["variant--light"]}>{headings}</div>
+        <div className={styles["variant--dark"]}>{headings}</div>
+      </div>
+    );
+  },
+  parameters: {
```

Comment:
should we skip snapshot testing for this one also? since it's skipped for `Text`

Reply:
Sounds good, will do